docs/content/developers/deploy_with_helm.md

```diff
@@ -0,0 +1,84 @@
+collection of templates for all the resources needed to deploy on k8s. ONOS Config
+provides a Helm chart for deploying a cluster for development and testing. In the
+future, this chart will be extended for production use.
+
+### Resources
+
+The Helm chart provides resources for deploying the config service and accessing
+it over the network, both inside and outside the k8s cluster:
+* `Deployment` - Provides a template for ONOS Config pods
+* `ConfigMap` - Provides test configurations for the application
+* `Service` - Exposes ONOS Config to other applications on the network
+* `Secret` - Provides TLS certificates for end-to-end encryption
+* `Ingress` - Optionally provides support for external load balancing
+
+### Local Deployment Setup
+
+To deploy the Helm chart locally, install [Kind] and [Helm]. On OSX, this can be done
+using [Brew]:
+
+```bash
+> go get sigs.k8s.io/kind
+> brew install kubernetes-helm
+```
+
+You will also need Docker to build and deploy an image locally.
+* Docker [installation instructions](https://docs.docker.com/v17.12/install/)
+
+
+Once Kind has been installed, start it with `kind create cluster`. 
+
+Once Kind has started, set your  environment to the Kubernetes cluster:
+
+```bash
+> export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+```
+### Deploy Atomix Controller
+
+The various `onos` services leverage Atomix as the distributed store for HA, scale and redundancy.
+The first thing that needs to be deployed in any `onos` deployment is the Atomix go controller.
+Fetch the `atomix-controller.yaml` file:
+```bash
+curl -o atomix-controller.yaml https://raw.githubusercontent.com/atomix/atomix-k8s-controller/master/deploy/atomix-controller.yaml
+```
+if you want to deploy the cluster in the non default namespace please edit `atomix-controller.yaml`, replacing `default`
+with your namespace.
+
+To then deploy the atomix controller issue:
+```bash
+kubectl create -f atomix-controller.yaml
+```
+with the correct return output being 
+```bash
+customresourcedefinition.apiextensions.k8s.io/partitionsets.k8s.atomix.io created
+customresourcedefinition.apiextensions.k8s.io/partitions.k8s.atomix.io created
+clusterrole.rbac.authorization.k8s.io/atomix-controller created
+clusterrolebinding.rbac.authorization.k8s.io/atomix-controller created
+serviceaccount/atomix-controller created
+deployment.apps/atomix-controller created
+service/atomix-controller created
+```
+If you watch the `pods` you should now see:
+```bash
+kubectl get pods --all-namespaces
+
+NAMESPACE     NAME                                         READY   STATUS    RESTARTS   AGE
+default       atomix-controller-b579b9f48-lgvxf            1/1     Running   0          152m
+```
+
+### Deploy services
+
+Once you have exported the `KUBECONFIG` flag you can start deploy `onos` services through helm charts.
+Please refer to each service's `deployment` file to get the exact command for each helm chart.
+An example can be found here:
```

Comment:
I assume you are still working on this part.

Reply:
done